src/test/java/com/softdevsix/ArgosApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package com.softdevsix;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ArgosApplicationTest {
+
+//  @Test
```

Comment:
Why did they comment on the test? If it is not necessary, it would be good if they eliminated it.

Reply:
There is minimum required by JaCoCo into the project configuration so this test is being covered due to its from project creation.